lib/utils/oclif-utils.ts

```diff
@@ -69,3 +69,17 @@ export function capitanoizeOclifUsage(
 		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, (match) => `<${match}>`)
 		.toLowerCase();
 }
+
+export async function getCommandsFromManifest() {
```

Comment:
NIT: These functions (`getCommandsFromManifest`, `getCommandIdsFromManifest`) are no longer async (`require` is synchronous). The processing of promises with async/await has a small performance penalty.